pytorch_lightning/loggers/tensorboard.py

```diff
@@ -115,13 +115,8 @@ def log_hyperparams(self, params: Union[Dict[str, Any], Namespace],
             )
         else:
             from torch.utils.tensorboard.summary import hparams
-            if metrics is None:
-                metrics = {}
-            exp, ssi, sei = hparams(sanitized_params, metrics)
-            writer = self.experiment._get_file_writer()
-            writer.add_summary(exp)
-            writer.add_summary(ssi)
-            writer.add_summary(sei)
+
+            self.experiment.add_hparams(sanitized_params, metrics)
```

Comment:
```suggestion
            if sanitized_params is not None and metrics is not None:
                self.experiment.add_hparams(sanitized_params, metrics)
```

Reply:
In the next version of pytorch hparam or metric equals None removed from doc and also crashes currently. 
https://pytorch.org/docs/master/tensorboard.html

Reply:
But this would be a breaking change because you can't log hparams without metrics taht way :)